raster-foundry/blasterjs
# Fix dependency issues

## Overview

- Combed through all dependencies to remove duplication and fix conflicts
  - Fixes warning of multiple instances of styled-components, which was breaking components utilizing `withTheme`
  - Fixes testBed running multiple instances of React when we ran `learn bootstrap`. This was preventing us from properly utilizing React Hooks within styled-components.
- Reverts the package version for both `docs-app` and `core` back to `1.0.0-alpha.3` (current)

### Checklist


### Upgrade instructions

If there are any of the following in this PR, provide proper instructions on how to upgrade:


## Notes
This does not attempt to fix the borked `Icon` component. That will come in a later PR.

## Testing Instructions

- `npm run clean`
- `npm install`
- Spin up docz to ensure it works correctly
- Spin up testBed to ensure it works correctly
